x-pack/plugins/monitoring/server/routes/api/v1/apm/instances.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Joi from 'joi';
+import { prefixIndexPattern } from '../../../../lib/ccs_utils';
+import { getApmClusterStatus } from './_get_apm_cluster_status';
+import { getApms } from '../../../../lib/apm/get_apms';
+import { handleError } from '../../../../lib/errors';
+
+export function apmInstancesRoute(server) {
+  /**
+   * Kibana listing (instances)
+   */
+  server.route({
+    method: 'POST',
+    path: '/api/monitoring/v1/clusters/{clusterUuid}/apm/instances',
```

Comment:
General question about the API path: Is everything `v1` at the moment or are there `v2` parts also in KB. Or is `v1`  just a convention on our side?

Reply:
The latter I think. I'm assuming it's safe guarding us against making breaking change (where we'd just do a `v2` for the breaking change)